src/app/modules/submission/multirespondent-submission/multirespondent-submission.service.ts

```diff
@@ -357,9 +465,38 @@ export const createMultiRespondentFormSubmission = ({
         mrfVersion,
       } = encryptedPayload
 
+      const nextStepNumber = 1 // since current step is 0
```

Comment:
CMIIW, this section ensures that moving forward MRF without subsequent email addresses will not be allowed to submit? -- was trying to figure out what this section of code was trying to achieve in relation to the send reminders feature.

Reply:
1. If that's the case, it will be valuable to note this change down in this PR (explain what changed).
2. Also that we've explicitly chosen to not pass form-level key as part of the reminder for security reasons. This will be valuable to note down in the PRD (under engineering / technical documentation).

Reply:
Hi @KenLSM, for the line highlighted, there is actually no change compared to previous before reminders and does not affect MRF without subsequent email addresses. 

The only change in the `createMultiRespondentFormSubmission` function is to add the storing of the nextStepWorkflowEmailAddresses into the metadata. 

Note that this function `retrieveWorkflowStepEmailAddresses` is not reading from the metadata, but rather from the `encryptedPayload.responses` directly.

Reply:
As for the PRD update, let me do so now